homework01/artur/vector.cc

```diff
@@ -0,0 +1,55 @@
+#include "vector.h"
+
+Vector::Vector()
+{
+}
```

Comment:
It is usually better to initialize things, but it's not wrong, it's an API choice.